source/GarminSDComms.mc

```diff
@@ -174,14 +173,14 @@ class GarminSDComms {
       }
     }
     lastOnSdStatusReceiveResponse = responseCode;
+    mStatusRequestInProgress = 0;
```

Comment:
I think this is just moving the re-setting of the flag from the start of the function to the end - putting it at the end means it will not be set if there is a crash in the main body of the function, so this is sensible - although a crash will kill the app anyway.